<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="3" ignored="0" total="8" passed="4">
  <reporter-output>
    <line>
      <![CDATA[<a href='E:\Photos\photo3.jpg' target='_blank'>Screenshot link </a> ]]>
    </line>
    <line>
      <![CDATA[<br>]]>
    </line>
    <line>
      <![CDATA[<a href='E:\Photos\photo3.jpg' target='_blank'><img height = 150 width = 200 src='E:\Photos\photo3.jpg'></a> ]]>
    </line>
    <line>
      <![CDATA[<a href='E:\Photos\photo3.jpg' target='_blank'>Screenshot link </a> ]]>
    </line>
    <line>
      <![CDATA[<br>]]>
    </line>
    <line>
      <![CDATA[<a href='E:\Photos\photo3.jpg' target='_blank'><img height = 150 width = 200 src='E:\Photos\photo3.jpg'></a> ]]>
    </line>
    <line>
      <![CDATA[<a href='E:\Photos\photo3.jpg' target='_blank'>Screenshot link </a> ]]>
    </line>
    <line>
      <![CDATA[<br>]]>
    </line>
    <line>
      <![CDATA[<a href='E:\Photos\photo3.jpg' target='_blank'><img height = 150 width = 200 src='E:\Photos\photo3.jpg'></a> ]]>
    </line>
  </reporter-output>
  <suite name="Functional Suite" duration-ms="250" started-at="2020-06-14T11:21:45 EEST" finished-at="2020-06-14T11:21:46 EEST">
    <groups>
      <group name="bvt">
        <method signature="TestCase3.fourthTest()[pri:4, instance:testcases.TestCase3@5c7fa833]" name="fourthTest" class="testcases.TestCase3"/>
      </group> <!-- bvt -->
      <group name="functional">
        <method signature="TestCase1.doLogin()[pri:2, instance:testcases.TestCase1@50c87b21]" name="doLogin" class="testcases.TestCase1"/>
        <method signature="TestCase1.doUserReg()[pri:1, instance:testcases.TestCase1@50c87b21]" name="doUserReg" class="testcases.TestCase1"/>
        <method signature="TestCase3.doUserReg()[pri:1, instance:testcases.TestCase3@5c7fa833]" name="doUserReg" class="testcases.TestCase3"/>
        <method signature="TestCase3.doLogin()[pri:2, instance:testcases.TestCase3@5c7fa833]" name="doLogin" class="testcases.TestCase3"/>
        <method signature="TestCase3.thirdTest()[pri:3, instance:testcases.TestCase3@5c7fa833]" name="thirdTest" class="testcases.TestCase3"/>
      </group> <!-- functional -->
      <group name="smoke">
        <method signature="TestCase2.validateTitles()[pri:0, instance:testcases.TestCase2@28feb3fa]" name="validateTitles" class="testcases.TestCase2"/>
        <method signature="TestCase3.doUserReg()[pri:1, instance:testcases.TestCase3@5c7fa833]" name="doUserReg" class="testcases.TestCase3"/>
        <method signature="TestCase3.doLogin()[pri:2, instance:testcases.TestCase3@5c7fa833]" name="doLogin" class="testcases.TestCase3"/>
        <method signature="TestCase3.thirdTest()[pri:3, instance:testcases.TestCase3@5c7fa833]" name="thirdTest" class="testcases.TestCase3"/>
      </group> <!-- smoke -->
    </groups>
    <test name="Login Test" duration-ms="63" started-at="2020-06-14T11:21:45 EEST" finished-at="2020-06-14T11:21:46 EEST">
      <class name="testcases.TestCase1">
        <test-method status="PASS" signature="createDBConnection()[pri:0, instance:testcases.TestCase1@50c87b21]" name="createDBConnection" is-config="true" duration-ms="0" started-at="2020-06-14T11:21:45 EEST" finished-at="2020-06-14T11:21:45 EEST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createDBConnection -->
        <test-method status="PASS" signature="doUserReg()[pri:1, instance:testcases.TestCase1@50c87b21]" name="doUserReg" duration-ms="0" started-at="2020-06-14T11:21:45 EEST" finished-at="2020-06-14T11:21:45 EEST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- doUserReg -->
        <test-method status="PASS" signature="launchBrowser()[pri:0, instance:testcases.TestCase1@50c87b21]" name="launchBrowser" is-config="true" duration-ms="0" started-at="2020-06-14T11:21:45 EEST" finished-at="2020-06-14T11:21:45 EEST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchBrowser -->
        <test-method status="PASS" signature="closeBrowser()[pri:0, instance:testcases.TestCase1@50c87b21]" name="closeBrowser" is-config="true" duration-ms="0" started-at="2020-06-14T11:21:45 EEST" finished-at="2020-06-14T11:21:45 EEST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
        <test-method status="PASS" signature="doLogin()[pri:2, instance:testcases.TestCase1@50c87b21]" name="doLogin" duration-ms="0" started-at="2020-06-14T11:21:46 EEST" finished-at="2020-06-14T11:21:46 EEST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- doLogin -->
        <test-method status="PASS" signature="closeBrowser()[pri:0, instance:testcases.TestCase1@50c87b21]" name="closeBrowser" is-config="true" duration-ms="0" started-at="2020-06-14T11:21:46 EEST" finished-at="2020-06-14T11:21:46 EEST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
        <test-method status="PASS" signature="closeDBConnection()[pri:0, instance:testcases.TestCase1@50c87b21]" name="closeDBConnection" is-config="true" duration-ms="0" started-at="2020-06-14T11:21:46 EEST" finished-at="2020-06-14T11:21:46 EEST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeDBConnection -->
        <test-method status="PASS" signature="launchBrowser()[pri:0, instance:testcases.TestCase1@50c87b21]" name="launchBrowser" is-config="true" duration-ms="0" started-at="2020-06-14T11:21:46 EEST" finished-at="2020-06-14T11:21:46 EEST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchBrowser -->
      </class> <!-- testcases.TestCase1 -->
    </test> <!-- Login Test -->
    <test name="Validate Title Test" duration-ms="16" started-at="2020-06-14T11:21:46 EEST" finished-at="2020-06-14T11:21:46 EEST">
      <class name="testcases.TestCase2">
        <test-method status="FAIL" signature="validateTitles()[pri:0, instance:testcases.TestCase2@28feb3fa]" name="validateTitles" duration-ms="0" started-at="2020-06-14T11:21:46 EEST" finished-at="2020-06-14T11:21:46 EEST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
expected [Yahoo.com] but found [Gmail.com],
image not present expected [false] but found [true],
text box not present expected [false] but found [true]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [Yahoo.com] but found [Gmail.com],
image not present expected [false] but found [true],
text box not present expected [false] but found [true]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at testcases.TestCase2.validateTitles(TestCase2.java:32)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:72)
at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[<a href='E:\Photos\photo3.jpg' target='_blank'>Screenshot link </a> ]]>
            </line>
            <line>
              <![CDATA[<br>]]>
            </line>
            <line>
              <![CDATA[<a href='E:\Photos\photo3.jpg' target='_blank'><img height = 150 width = 200 src='E:\Photos\photo3.jpg'></a> ]]>
            </line>
          </reporter-output>
        </test-method> <!-- validateTitles -->
      </class> <!-- testcases.TestCase2 -->
    </test> <!-- Validate Title Test -->
    <test name="Dependency Test" duration-ms="16" started-at="2020-06-14T11:21:46 EEST" finished-at="2020-06-14T11:21:46 EEST">
      <class name="testcases.TestCase3">
        <test-method status="SKIP" signature="doLogin()[pri:2, instance:testcases.TestCase3@5c7fa833]" name="doLogin" duration-ms="0" started-at="2020-06-14T11:21:46 EEST" depends-on-methods="testcases.TestCase3.doUserReg" finished-at="2020-06-14T11:21:46 EEST">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method TestCase3.doLogin()[pri:2, instance:testcases.TestCase3@5c7fa833] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method TestCase3.doLogin()[pri:2, instance:testcases.TestCase3@5c7fa833] depends on not successfully finished methods
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:101)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:72)
at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- doLogin -->
        <test-method status="PASS" signature="thirdTest()[pri:3, instance:testcases.TestCase3@5c7fa833]" name="thirdTest" duration-ms="0" started-at="2020-06-14T11:21:46 EEST" depends-on-methods="testcases.TestCase3.doUserReg" finished-at="2020-06-14T11:21:46 EEST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- thirdTest -->
        <test-method status="FAIL" signature="doUserReg()[pri:1, instance:testcases.TestCase3@5c7fa833]" name="doUserReg" duration-ms="0" started-at="2020-06-14T11:21:46 EEST" finished-at="2020-06-14T11:21:46 EEST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[User is not registered successfully]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: User is not registered successfully
at org.testng.Assert.fail(Assert.java:97)
at testcases.TestCase3.doUserReg(TestCase3.java:16)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:72)
at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[<a href='E:\Photos\photo3.jpg' target='_blank'>Screenshot link </a> ]]>
            </line>
            <line>
              <![CDATA[<br>]]>
            </line>
            <line>
              <![CDATA[<a href='E:\Photos\photo3.jpg' target='_blank'><img height = 150 width = 200 src='E:\Photos\photo3.jpg'></a> ]]>
            </line>
          </reporter-output>
        </test-method> <!-- doUserReg -->
        <test-method status="PASS" signature="fourthTest()[pri:4, instance:testcases.TestCase3@5c7fa833]" name="fourthTest" duration-ms="0" started-at="2020-06-14T11:21:46 EEST" finished-at="2020-06-14T11:21:46 EEST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- fourthTest -->
      </class> <!-- testcases.TestCase3 -->
    </test> <!-- Dependency Test -->
    <test name="Failure Test" duration-ms="0" started-at="2020-06-14T11:21:46 EEST" finished-at="2020-06-14T11:21:46 EEST">
      <class name="testcases.TestFailure">
        <test-method status="PASS" signature="setUp()[pri:0, instance:testcases.TestFailure@cb5822]" name="setUp" is-config="true" duration-ms="19" started-at="2020-06-14T11:21:45 EEST" finished-at="2020-06-14T11:21:45 EEST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="doLogin()[pri:0, instance:testcases.TestFailure@cb5822]" name="doLogin" duration-ms="0" started-at="2020-06-14T11:21:46 EEST" finished-at="2020-06-14T11:21:46 EEST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Failing the login test]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Failing the login test
at org.testng.Assert.fail(Assert.java:97)
at testcases.TestFailure.doLogin(TestFailure.java:12)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:72)
at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[<a href='E:\Photos\photo3.jpg' target='_blank'>Screenshot link </a> ]]>
            </line>
            <line>
              <![CDATA[<br>]]>
            </line>
            <line>
              <![CDATA[<a href='E:\Photos\photo3.jpg' target='_blank'><img height = 150 width = 200 src='E:\Photos\photo3.jpg'></a> ]]>
            </line>
          </reporter-output>
        </test-method> <!-- doLogin -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:testcases.TestFailure@cb5822]" name="tearDown" is-config="true" duration-ms="0" started-at="2020-06-14T11:21:46 EEST" finished-at="2020-06-14T11:21:46 EEST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- testcases.TestFailure -->
    </test> <!-- Failure Test -->
  </suite> <!-- Functional Suite -->
</testng-results>
